tools/clang/test/CodeGenSPIRV/sm6.quad-any-all.hlsl

```diff
@@ -0,0 +1,36 @@
+// RUN: %dxc -T cs_6_0 -E main -fspv-target-env=vulkan1.1 -fcgl  %s -spirv | FileCheck %s --check-prefixes=CHECK,QUAD
+// RUN: %dxc -T cs_6_0 -E main -fspv-target-env=vulkan1.1 -fspv-extension=SPV_KHR_16bit_storage -fcgl  %s -spirv | FileCheck %s --check-prefixes=CHECK,NOQUAD
```

Comment:
could we do a test with vulkan1.0 and check for the error? I think we can simply add another run command.

Reply:
Done.